src/processor.cpp

```diff
@@ -602,19 +622,29 @@ class _Processor
             case Opcode::TIMESTAMP:
                 timestamp();
                 break;
-            case Opcode::DIFFICULTY:
-                difficulty();
+            case Opcode::PREVRANDAO:
+                prevrandao();
```

Comment:
```suggestion
                prevrandao();
```
is difficult?

Reply:
Renaming: The DIFFICULTY (0x44) opcode SHOULD further be renamed to PREVRANDAO (0x44).

Reply:
Since the contract execute on off-chain, it is not necessary for some opcode, such as PREVERANDAO, BASEFEE, GASLIMIT, etc.

Reply:
Done